lib/index.js

```diff
@@ -513,8 +524,11 @@ Record.prototype.read = function (name) {
     return Promise.resolve()
   }
   var args = [
-    [this.id]
+    ['id', '=', this.id], null, null, null
   ]
+  if (this.session.party) {
+    args[0] = alterDomain(this.session.party, this.model.name, args[0])
```

Comment:
ça devrait juste prendre en argument this.session, et si y a des restrictions de domaines défini dans la session, les appliquer.